melange-react-intl.opam

```diff
@@ -7,18 +7,18 @@ license: "MIT"
 homepage: "https://github.com/ahrefs/melange-react-intl-ppx"
 bug-reports: "https://github.com/ahrefs/melange-react-intl-ppx/issues"
 
-synopsis: "PPX simplifying usage of react-intl internationalization library"
+synopsis: "react-intl internationalization library binding and PPX reduce messages boilerplate code"
 
 depends: [
   "ocaml" {>= "4.14.0"}
   "dune" {>= "3.0"}
   "ppxlib" {>= "0.21.0"}
   "reason" {>= "3.7.0"}
   "re" {= "1.10.4"}
+  "reason-react" {= "0.11.0+20230626"}
```

Comment:
This will not work. There is no such version published in the public opam repository.

I would suggest to leave it unset for now:

```suggestion
  "reason-react"
```

Reply:
Updated in this commit: https://github.com/ahrefs/melange-react-intl-ppx/commit/c41de8653cc4932c4709ff6dece6bdbe06fca418

Reply:
Thanks!